### PR TITLE
create style guide, HomeView component and Sponsor component

### PR DESCRIPTION
**Pull request by markdreed512**
This is just a draft, we can discuss at the next meeting and decide if we should change or add anything else.

### CODE REVIEW

**Comment by markdreed512** [comment]
Can I get write access so I can merge pull requests?

**Comment by I-MUST-DANCE** [comment]
> Can I get write access so I can merge pull requests?

I thought you were added as a community member for the repo. Last week I checked with Valentine and he said the email invite was still pending?

File: src/App.css [hunk 1]
```diff
@@ -1,45 +1,38 @@
-#root {
-  max-width: 1280px;
-  margin: 0 auto;
-  padding: 2rem;
-  text-align: center;
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@0,100..900;1,100..900&display=swap');
```
Comment (I-MUST-DANCE) [inline src/App.css:1]: Great for getting us off the ground, especially when much of the figma design needs to be scraped for font family's and values. When we get closer to production, we can optimize by moving to a cdn or self-hosting our font files.

File: src/components/Sponsors/Sponsors.jsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import { uid } from 'uid'
+import SponsorCard from '../SponsorCard/SponsorCard'
+import './Sponsors.css'
+
+const sponsors = [
+    {
+        name: "Wix Studio",
+        type: "Financial Sponsor",
+        description: "Lorem ipsum dolor sit amet, consectetur adipisicing elit. ",
+        logo: "https://static.wixstatic.com/media/0d6674_b39c8ddaafbd4b8dbb459a4221ee8227~mv2.png/v1/fill/w_524,h_519,al_c,q_85,usm_0.66_1.00_0.01,enc_auto/Frame%2010000017556.png",
```
Comment (I-MUST-DANCE) [inline src/components/Sponsors/Sponsors.jsx:11]: We likely want to use the .png from /assets, but that can be fixed later when the filler gets replaced with real copy.

**Pull request merged**